Lib/test/test_doctest5.py

```diff
@@ -0,0 +1,33 @@
+"""A module to test whether doctest works properly with
+un-unwrappable functions.
+
+See: https://bugs.python.org/issue35753
+"""
```

Comment:
Please add the test to test_doctest, don't create a new file.

Reply:
Done.  Thank you.